client/src/components/HealthLogToggle.jsx

```diff
@@ -19,13 +19,12 @@ const useStyles = makeStyles({
 });
 
 const HealthLogToggle = props => {
-  const { toggleValue, setToggleValue, setDetailData, observations } = props;
+  const { toggleValue, setToggleValue } = props;
   const classes = useStyles();
 
   const onShowMeMoreClick = useCallback(() => {
-    setDetailData(observations);
```

Comment:
![Screen Shot 2020-05-06 at 10 03 51 PM](https://user-images.githubusercontent.com/15807680/81246432-95b26280-8fe5-11ea-906a-e09794933fde.png)
But in show me more we've used a ternary to say if we haven't set some subset of observations as detailData then we will show all the observations.  So setting detailData to the whole observations list is redundant, isn't it?  We only need detailData to be some subset of observations.  I'm actually not clear on why the whole detailData store even needs to exist.  detailData is only ever either a) the whole observations list which already exists or b) some subset of the observations list which could be achieved just by filtering the observations list based on the existence of a selected date.  The problem I was presented with in this case was that because we were using detailData in the component instead of observations, converting the temperature values on the observations list in the redux store did not transform them in this component.  Cutting out a redundant copy of the list seemed better than having to transform both lists.

Reply:
This `setDetailData` call is handling the case where a user swipes on a particular observation on the calendar screen, opening the "Show Me More" screen with only that observation's data showing. Then, the user navigates back to the calendar screen and clicks the "Show Me More" button.

With this function call removed, the data for that single observation will be shown again. We want it to show all the observations this time, so we set the detail data to `observations`. A possible solution to the problem you're facing is calling `setDetailData([])` here, but you should test it out to confirm

Reply:
@acthelemann As I work to address this, I'm curious to know, does this swipe on an observation action work in the way you describe currently?  It doesn't seem to work for me running this locally, which maybe was why I couldn't see any negative impact of the approach I took.  When I swipe on an observation nothing happens at all -- no navigating to the show me more screen, and no redux action dispatched.  I don't think anything I did is the reason it isn't working for me locally, I assume that's just where we are, but I want to account for the possibility that I broke something.

Reply:
The first question is are you swiping correctly. People have expressed issues that the action of swiping is difficult. You should be able to.

Test it out on the beta site. You definitely can there.

https://coronatrackerbeta.com/